data/rulesets.ts

```diff
@@ -2033,7 +2032,7 @@ export const Rulesets: {[k: string]: FormatData} = {
 				newPoke.position = newPos;
 				source.side.pokemon[newPos] = newPoke;
 				this.add('poke', source.side.pokemon[newPos].side.id, source.side.pokemon[newPos].details, '');
-				this.add('-message', `${target.name} was captured by ${newPoke.side.name}!`);
+				this.add('-message', `${target.name} was captured by ${Utils.escapeHTML(newPoke.side.name)}!`);
```

Comment:
Only raw messages need to be escaped by the server. This is not, so it's automatically escaped client-side.